File: poslovna/src/main/java/com/poslovna/domain/StavkaCenovnika.java
package com.poslovna.domain;

import javax.persistence.GeneratedValue;
import javax.persistence.Id;

public class StavkaCenovnika {
	
	@Id
	@GeneratedValue
	private Long id;
	
	private double cena;
	
	private Proizvod proizvod;

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public double getCena() {
		return cena;
	}

	public void setCena(double cena) {
		this.cena = cena;
	}
	
	
	
	

}

File: frontend-poslovna/src/app/components/models/stavkacenovnika.ts
import { Proizvod } from './proizvod';

export class StavkaCenovnika {
  procenat: number;
  proizvod: Proizvod;
  cena: number;
  id: number;
  constructor(
      cena: number,
      proizvod: Proizvod,
      procenat1: number,
      id?: number
  ) {
    this.procenat = procenat1;
    this.proizvod = proizvod;
    this.cena = cena;
    this.id = id;
}
}